src/main/kotlin/com/mparticle/kits/ApptentiveKit.kt

```diff
@@ -159,134 +194,83 @@ class ApptentiveKit : KitIntegration(), KitIntegration.EventListener, CommerceLi
-                        event.customAttributeStrings,
-                        apptentiveCommerceData
-                    )
-                    val messages = LinkedList<ReportingMessage>()
-                    messages.add(ReportingMessage.fromEvent(this, event))
-                    return messages
-                }
-            } catch (jse: JSONException) {
-            }
-        }
-        return emptyList()
+    override fun onLogoutCompleted(
+        mParticleUser: MParticleUser?,
+        identityApiRequest: FilteredIdentityApiRequest?
+    ) {
+        setUserIdentity(mParticleUser)
     }
 
-    private fun setTransactionAttributes(
-        transactionAttributes: TransactionAttributes,
-        eventActionAttributes: HashMap<String, String>,
-        event: CommerceEvent
-    ): CommerceExtendedData {
-        val apptentiveCommerceData = CommerceExtendedData()
-        val transactionId = transactionAttributes.id
-        if (!KitUtils.isEmpty(transactionId)) {
-            apptentiveCommerceData.setId(transactionId)
-        }
-        val transRevenue = transactionAttributes.revenue
-        if (transRevenue != null) {
-            apptentiveCommerceData.setRevenue(transRevenue)
-        }
-        val transShipping = transactionAttributes.shipping
-        if (transShipping != null) {
-            apptentiveCommerceData.setShipping(transShipping)
-        }
-        val transTax = transactionAttributes.tax
-        if (transTax != null) {
-            apptentiveCommerceData.setTax(transTax)
-        }
-        val transAffiliation = transactionAttributes.affiliation
-        if (!KitUtils.isEmpty(transAffiliation)) {
-            apptentiveCommerceData.setAffiliation(transAffiliation)
-        }
-        var transCurrency =
-            eventActionAttributes[CommerceEventUtils.Constants.ATT_ACTION_CURRENCY_CODE]
-        if (KitUtils.isEmpty(transCurrency)) {
-            transCurrency = CommerceEventUtils.Constants.DEFAULT_CURRENCY_CODE
-        }
-        apptentiveCommerceData.setCurrency(transCurrency)
-
-        // Add each item
-        val productList = event.products
-        if (productList != null) {
-            for (product in productList) {
-                val item = CommerceExtendedData.Item()
-                item.setId(product.sku)
-                item.setName(product.name)
-                item.setCategory(product.category)
-                item.setPrice(product.unitPrice)
-                item.setQuantity(product.quantity)
-                item.setCurrency(transCurrency)
-                apptentiveCommerceData.addItem(item)
-            }
-        }
-        return apptentiveCommerceData
+    override fun onModifyCompleted(
+        mParticleUser: MParticleUser?,
+        identityApiRequest: FilteredIdentityApiRequest?
+    ) {
+        setUserIdentity(mParticleUser)
     }
 
-    //region Notifications
-    override fun onReceiveNotification(notification: ApptentiveNotification) {
```

Comment:
Is this already handled elsewhere?

Reply:
Yes, the new SDK doesn't use the legacy notification system. The mParticle ID is being set in the Apptentive register listener and updated in the `IdentityListener` methods